package.json

```diff
@@ -79,21 +70,14 @@
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
     "//[ Watches ]//": "",
-    "watch:font": "onchange $npm_package_config_src_font'/**/**' -- npm-run-all sync:font reload",
-    "watch:font:debug": "onchange $npm_package_config_src_font'/**/**' -- npm-run-all sync:font",
-    "watch:img": "onchange $npm_package_config_src_img'/**/**' -- npm-run-all sync:img reload",
-    "watch:img:debug": "onchange $npm_package_config_src_img'/**/**' -- npm-run-all sync:img",
-    "watch:js": "onchange $npm_package_config_dest_js'/**/**' -- npm-run-all -p reload lint:js",
-    "watch:css": "onchange $npm_package_config_dest_css -- npm-run-all -p reload lint:css",
+    "watch:js": "onchange $npm_package_config_dest_js'/**/**' -- npm run lint:js",
+    "watch:css": "onchange $npm_package_config_dest_css -- npm run lint:css",
     "watch": "npm-run-all -p watch:*",
-    "//[ Syncs ]//": "",
-    "sync:font": "rsync -rtvu --delete $npm_package_config_src_font/ $npm_package_config_dest_font/",
-    "sync:img": "rsync -rtvu --delete $npm_package_config_src_img/ $npm_package_config_dest_img/",
-    "sync": "npm-run-all -p sync:*",
-    "//[ Tasks ]//": "",
-    "build": "npm-run-all clean create_dirs sync compile:dev",
-    "build:prod": "npm-run-all clean create_dirs sync compile:prod",
-    "start": "npm-run-all clean create_dirs sync compile:dev -p compile:watch watch serve --silent"
+    "//[ Build tasks ]//": "",
+    "build": "npm run clean && webpack --mode development --progress",
+    "build:prod": "npm run clean && webpack --mode production",
+    "start": "npm-run-all clean -p compile:watch watch --silent",
+    "start:reload": "npm-run-all clean -p compile:reload watch --silent"
```

Comment:
@nicklee I've asked for your review on this because I've made some quite large edits to some FE tooling I'm not 100% confident about. When npm runs in the container I'm seeing a lot of things like:

```
 229:29  ✖  Unexpected duplicate name monospace                                                                                                     font-family-no-duplicate-names
 309:5   ✖  Unexpected vendor-prefix "-webkit-appearance"                                                                                           property-no-vendor-prefix
 395:5   ✖  Unexpected vendor-prefix "-webkit-appearance"                                                                                           property-no-vendor-prefix
 405:5   ✖  Unexpected vendor-prefix "-webkit-appearance"                                                                                           property-no-vendor-prefix
 412:3   ✖  Unexpected vendor-prefix "::-webkit-input-placeholder"                                                                                  selector-no-vendor-prefix
 423:5   ✖  Unexpected vendor-prefix "-webkit-appearance"                                                                                           property-no-vendor-prefix
```
Although it might be somewhat expected